source/client/renderer/GameRenderer.cpp

```diff
@@ -680,7 +863,7 @@ void GameRenderer::render(float f)
 	debugText << "ReMinecraftPE " << m_pMinecraft->getVersionString();
 	debugText << "\n" << m_shownFPS << " fps, " << m_shownChunkUpdates << " chunk updates";
 
-	if (m_pMinecraft->getOptions()->m_bDebugText)
+	if (m_pMinecraft->getOptions()->m_bDebugText && !m_pMinecraft->getOptions()->m_bDontRenderGui)
```

Comment:
finally! actually it's not part of isometric renderer but anyway

Reply:
To be honest this condition is redundant, as if `m_bDontRenderGui` is true, there's an early return